ivy/functional/ivy/activations.py

```diff
@@ -609,3 +610,69 @@ def mish(
     }
     """
     return current_backend(x).mish(x, out=out)
+
+
+@handle_exceptions
+@handle_nestable
+@handle_array_like_without_promotion
+@handle_out_argument
+@to_native_arrays_and_back
+@handle_array_function
+def elu(
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    alpha: float = 1.0,
+    out: Optional[ivy.Array] = None,
+) -> ivy.Array:
+    """
+    Apply the elu unit function element-wise.
+
+    Parameters
+    ----------
+    x
+        Input array.
+    alpha
+        sclaer multiple of exponent slop of negative input.
```

Comment:
There is a typo here in scalar and slope would you look into the rest of docstring

Reply:
i have corrected this typo in doc string you can check it now.

thanks